src/Sentry/Laravel/Features/LivewirePackageIntegration.php

```diff
@@ -125,6 +202,12 @@ private function updateTransactionName(string $componentName): void
         $transaction->getMetadata()->setSource(TransactionSource::custom());
 
         Integration::setTransaction($transactionName);
+
+        $hub->configureScope(function (Scope $scope) {
+            $livewireManager = $this->container()->make(LivewireManager::class);
+
+            $scope->setTag('livewire.original_url', $livewireManager->originalUrl());
```

Comment:
SDKs should not send any tags by default. Is this something a user needs to be able to search for?

Reply:
@cleptric Livewire sends state updates to a global Livewire route (`/livewire/update`) and is not very helpful to find out the exact location of an issue. The original url helps to find out on what page the issue occurred. We already improved this issue by renaming the transaction to include the Livewire component. Livewire components can be used on many pages so adding this tag makes it much easier to track down the origin of the issue.

Reply:
If the transaction name is already fixed, the `transaction` value of an error should reflect this as well. So a user will see issues grouped on a route basis afaik.

Reply:
We've decided to remove this for now and see in a follow up if we can add this parameter somewhere and make it searchable still. But we want to ship this so we'll leave it out for now.